modules/http4s/src/main/scala/com/evolutiongaming/smetrics/Http4sMetricsOps.scala

```diff
@@ -0,0 +1,116 @@
+package com.evolutiongaming.smetrics
+
+import cats.effect._
+import cats.implicits._
+import com.evolutiongaming.smetrics.MetricsHelper._
+import org.http4s.metrics.TerminationType.{Abnormal, Error, Timeout}
+import org.http4s.metrics.{MetricsOps, TerminationType}
+import org.http4s.{Method, Status}
+
+object Http4sMetricsOps {
+
+  def of[F[_] : Sync](
+    cr: CollectorRegistry[F],
```

Comment:
please rename cr -> registry or collectorRegistry, to be consistent with other places

Reply:
fixed